run-locally.sh

```diff
@@ -0,0 +1 @@
+java -jar ./target/jpo-sdw-depositor-0.01-SNAPSHOT.jar
```

Comment:
Is this script used anywhere?

Reply:
This file isn't being used. It's meant as a convenience script for developers. Should we remove it?

Reply:
Do you think we need the ability to run the app as a jar locally? If a dev wants to test the app, I think they'd either launch it from the IDE, or run it in a docker container.

Reply:
It's not necessary, no. This file has been removed.